packages/ossa/src/utils/index.ts

```diff
@@ -22,9 +22,14 @@ export function deprecatedProp<T = unknown, P = T>(newProp: T, oldProp: P, warin
     newPropName: string;
     oldPropName: string;
     moduleName: string;
+    defaultVal?: T
 }): T | P {
     if(typeof oldProp !== 'undefined' && waringInfo && waringInfo.oldPropName && process.env.NODE_ENV === "development") {
         warnDeprecatedProp(waringInfo.moduleName, waringInfo.newPropName, waringInfo.oldPropName);
     }
-    return newProp ?? oldProp;
+    const res = newProp ?? oldProp;
+    if(res === undefined && waringInfo?.defaultVal !== undefined) {
+        return waringInfo?.defaultVal;
+    }
+    return  res;
```

Comment:
```suggestion
return newProp ?? oldProp ?? waringInfo?.defaultVal
```
这样是不是更简洁些？

另外，`defaultVal` 放在 `waringInfo` 中传入，从语义上是不是不太对？

Reply:
done